src/modules/game/BatteryAbsoluteTime.js

```diff
@@ -2,7 +2,7 @@ require("../../_core/HFX");
 class BatteryAbsoluteTime extends HFX.Feature {
   constructor() {
     super({
-      section: "game",
+      section: HFX.Section.Game,
```

Comment:
For the sake of the bug fix and this PR, this is excellent and fixes the problem.

However, in the future a module might be able to run on two specific pages.
For example
game.php
profile.php

The way you modeled this section logic, would it be possible for this section constructure to be given an array of Sections, like this:
section: [HFX.Section.Game, HFX.Section.Profile]

Reply:
It seems like it would be easiest to modify the Section constructor to take multiple locations. So, it would still be one section, but runnable on multiple pages.

Having multiple sections doesn't really work well with how settings are stored by the section name.